invenio_communities/views/communities.py

```diff
@@ -187,6 +188,8 @@ def communities_settings(pid_value, community, community_ui):
     if type(max_size) is int and max_size > 0:
         logo_size_limit = max_size
 
+    members_count = MemberModel.count_members(community_id=community.id)
```

Comment:
This is querying directly the data layer in the presentation layer, it doesn't feel right.  

I suggest for this information to be hooked somewhere else (either service layer or maybe in the UISerializer?). The community is read from the service and then serialized [here](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/views/decorators.py#L28)

Reply:
I agree, this is not following the architecture of application layers, see https://inveniordm.docs.cern.ch/develop/architecture/software/#service-layer